holidays/au.yaml

```diff
@@ -0,0 +1,141 @@
+---
+
+_nominatim_url: 'https://nominatim.openstreetmap.org/reverse?format=json&lat=-35.2809&lon=149.1300&zoom=16&addressdetails=1&accept-language=en'
+
+# https://en.wikipedia.org/wiki/Public_holidays_in_Australia
+# Partial holidays excluded
+PH:
+  'New Years Day': [1, 1]
+  'Australia Day': [1, 26]
+  'Good Friday': [easter, -2]
+  'Easter Monday': [easter, 1]
+  'ANZAC Day': [4, 25]
+  'Christmas Day': [12, 25]
+  'Boxing Day': [12, 26]
+
+'Australian Capital Territory':
+  _state_code: act
+  _nominatim_url: 'https://nominatim.openstreetmap.org/reverse?format=json&lat=-35.2809&lon=149.1300&zoom=16&addressdetails=1&accept-language=en'
+  PH:
+    'New Years Day': [1, 1]
+    'Australia Day': [1, 26]
+    'Canberra Day': [firstMarchMonday, 7]
+    'Good Friday': [easter, -2]
+    'Easter Saturday': [easter, -1]
+    'Easter Sunday': [easter, 0]
+    'Easter Monday': [easter, 1]
+    'ANZAC Day': [4, 25]
+    'Queens Birthday': [firstJuneMonday, 7]
+    'Family and Community Day': [lastSeptemberMonday, -7]
+    'Labour Day': [firstOctoberMonday, 0]
+    'Christmas Day': [12, 25]
+    'Boxing Day': [12, 26]
+
+'New South Wales':
+  _state_code: nsw
+  _nominatim_url: 'https://nominatim.openstreetmap.org/reverse?format=json&lat=-33.8688&lon=151.2093&zoom=16&addressdetails=1&accept-language=en'
+  PH:
+    'New Years Day': [1, 1]
+    'Australia Day': [1, 26]
+    'Good Friday': [easter, -2]
+    'Easter Saturday': [easter, -1]
+    'Easter Sunday': [easter, 0]
+    'Easter Monday': [easter, 1]
+    'ANZAC Day': [4, 25]
+    'Queens Birthday': [firstJuneMonday, 7]
+    'Labour Day': [firstOctoberMonday, 0]
+    'Christmas Day': [12, 25]
+    'Boxing Day': [12, 26]
+
+'Northern Territory':
+  _state_code: nt
+  _nominatim_url: 'https://nominatim.openstreetmap.org/reverse?format=json&lat=-12.4634&lon=130.8456&zoom=16&addressdetails=1&accept-language=en'
+  PH:
+    'New Years Day': [1, 1]
+    'Australia Day': [1, 26]
+    'Good Friday': [easter, -2]
+    'Easter Saturday': [easter, -1]
+    'Easter Monday': [easter, 1]
+    'ANZAC Day': [4, 25]
+    'May Day': [firstMayMonday, 0]
+    'Queens Birthday': [firstJuneMonday, 7]
+    'Picnic Day': [firstAugustMonday, 0]
+    'Christmas Day': [12, 25]
+    'Boxing Day': [12, 26]
+
+Queensland:
+  _state_code: qld
+  _nominatim_url: 'https://nominatim.openstreetmap.org/reverse?format=json&lat=-27.4698&lon=153.0251&zoom=16&addressdetails=1&accept-language=en'
+  PH:
+    'New Years Day': [1, 1]
+    'Australia Day': [1, 26]
+    'Good Friday': [easter, -2]
+    'Easter Saturday': [easter, -1]
+    'Easter Monday': [easter, 1]
+    'ANZAC Day': [4, 25]
+    'Labour Day': [firstMayMonday, 0]
+    'Queens Birthday': [firstOctoberMonday, 7]
+    'Christmas Day': [12, 25]
+    'Boxing Day': [12, 26]
+  
```

Comment:
Note that many well maintained projects have automated tests which are run against each PR. Refer to https://travis-ci.org/opening-hours/opening_hours.js/jobs/279563412#L645. Trailing whitespace is evil :) I fixed it for ya :)